src/sentry/api/serializers/rest_framework/base.py

```diff
@@ -65,7 +65,7 @@ class CamelSnakeModelSerializer(ModelSerializer):
     def __init__(self, instance=None, data=empty, **kwargs):
         if data is not empty:
             data = convert_dict_key_case(data, camel_to_snake_case)
-        return super().__init__(instance=instance, data=data, **kwargs)
+        super().__init__(instance=instance, data=data, **kwargs)
```

Comment:
@wedamija can you confirm if is this right?

Reply:
Yeah this is fine, not sure why I had a return there